npm/angular/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "private": true,
+  "name": "@cypress/angular",
+  "version": "0.0.0-development",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc --project tsconfig.lib.json",
+    "watch": "tsc --project tsconfig.lib.json -w",
+    "cy:open": "node ../../scripts/cypress.js open-ct",
+    "cy:run": "node ../../scripts/cypress.js run-ct",
+    "ng": "ng",
+    "app-start": "ng serve",
+    "app-build": "ng build",
+    "test": "yarn cy:run",
+    "test-ci": "yarn cy:run"
+  },
+  "dependencies": {
+    "@cypress/mount-utils": "0.0.0-development",
+    "debug": "4.3.2"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "0.1102.12",
+    "@angular-eslint/eslint-plugin": "4.2.0",
+    "@angular/animations": "11.2.13",
+    "@angular/cdk": "11.2.12",
+    "@angular/cli": "11.2.12",
+    "@angular/common": "11.2.13",
+    "@angular/compiler": "11.2.13",
+    "@angular/compiler-cli": "11.2.13",
+    "@angular/core": "11.2.13",
+    "@angular/forms": "11.2.13",
+    "@angular/language-service": "11.2.13",
+    "@angular/localize": "11.2.13",
+    "@angular/material": "11.2.12",
+    "@angular/platform-browser": "11.2.13",
+    "@angular/platform-browser-dynamic": "11.2.13",
+    "@angular/router": "11.2.13",
+    "@cypress/angular": "file:.",
+    "@cypress/code-coverage": "3.9.5",
+    "@cypress/webpack-dev-server": "1.1.4",
+    "@cypress/webpack-preprocessor": "5.7.0",
+    "@types/cypress-image-snapshot": "3.1.5",
+    "@types/node": "8.10.66",
+    "angular-router-loader": "0.8.5",
+    "angular2-template-loader": "0.6.2",
+    "codelyzer": "6.0.2",
+    "core-js": "3.12.1",
+    "cypress": "7.1.0",
```

Comment:
Shouldn't this be local Cypress?

Reply:
Yes
the test script uses local but I will remove this dependency

Reply:
Done.